common/frame.go

```diff
@@ -1113,7 +1107,7 @@ func (f *Frame) innerHTML(selector string, opts *FrameInnerHTMLOptions) (string,
 		return "", fmt.Errorf("unexpected type %T", v)
 	}
 
-	return gv.ToString().String(), nil
+	return gv.String(), nil
```

Comment:
Are we sure these are equivalent? The tests pass, so likely yes, but I'm wondering why the previous usage is common.

Reply:
I made a little research on this and here are my findings. For example, `Frame.Content()` uses `ExecutionContext.eval` to get a goja value:

```
// remote_object.go
func valueFromRemoteObject(...) {
    ...
    return k6ext.Runtime(ctx).ToValue(val)
}
```

The above function returns the value as `goja.asciiString`. `asciiString.ToString()` returns itself. Whereas, `asciiString.String()` returns itself as `string` as its underlying type is `string`. So `.ToString().String()` == `String()`.